data/wanderer/wanderer ships.txt

```diff
@@ -402,6 +420,9 @@ ship "Derecho"
 		"outfit space" 796
 		"weapon capacity" 342
 		"engine capacity" 164
+		"reverse thruster slot" 1
+		"steering slot" 1
+		"thruster slot" 1
```

Comment:
The Derecho is one of the stronger contenders for additional slots for thruster and/or steering (like what you've given the Peregrine). It's specified as being designed for speed, and with Wanderer engines it can only use 112 of its 164 engine space if limited to a single slot of each type (in comparison, the Peregrine can use 131 of its 171 engine space with only one Remnant thruster and one Remnant steering - a larger portion, and less space left over, and the Remnant also have access to an afterburner that can use some of that space too.

Reply:
I will consider it, but I think that's stretching the case a bit:

> Like a latent gene waiting to express itself, the knowledge of how to wage war in space has been passed down through hundreds of generations of Wanderers. Based on one of their ancient designs, the Derecho has a mixture of agility and raw firepower that is well-suited to their present, rapidly changing situation.

That doesn't sound like "Built for speed" to me, that sounds like a ship intended to be a balanced all-round combatant. Especially since this in direct contrast to the preceding Heavy Warship, which was explicitly designed as a slow-moving face-tank.

> The Hurricane is not built for speed, elegance, or versatility, but for one simple purpose: to be durable enough to serve as a rearguard, absorbing an enemy's attacks for long enough to allow the fleet it is guarding to escape.